pkg/kubectl/kubectl.go

```diff
@@ -142,6 +142,30 @@ var ResourcesShortcutStatic = []ResourceShortcuts{
 		ShortForm: schema.GroupResource{Group: "extensions", Resource: "psp"},
 		LongForm:  schema.GroupResource{Group: "extensions", Resource: "podSecurityPolicies"},
 	},
+	{
+		ShortForm: schema.GroupResource{Group: "apps.openshift.io", Resource: "dc"},
```

Comment:
with shortcuts in discovery, do we still need this?

Reply:
> with shortcuts in discovery, do we still need this?

@mfojtik we support one level of skew.  I think we supported shortnames in kube 1.7, we just need to make sure we've wired them.

Reply:
> @mfojtik we support one level of skew. I think we supported shortnames in kube 1.7, we just need to make sure we've wired them.

Here https://github.com/openshift/origin/pull/16823

Reply:
thanks! i will drop this carry